### ISSUE TITLE
Error: Next REST Framework config not found. Initialize a docs handler to generate the OpenAPI spec.

### ISSUE THREAD

**Issue by WillGarman** [blomqma/next-rest-framework#123]
I am using Windows 11 and even when using the `next-rest-framework` example repository, after going through the installations. When I run
- `pnpm run generate` (from the [package.json](https://github.com/blomqma/next-rest-framework/blob/main/apps/example/package.json))
- `npx next-rest-framework generate`

Any sort of generate command (Including in my personal project), fails with the error:

```
Compiling endpoints...
Generating OpenAPI spec...
Next REST Framework config not found. Initialize a docs handler to generate the OpenAPI spec.
```

- I have the docs handler file setup at `/api/route.ts` like in the documentation (also tried the [example repo](https://github.com/blomqma/next-rest-framework) and it didn't work

I also tried to setting the api path using:
`next-rest-framework generate --configPath /api` 
and got this error 
```
Compiling endpoints...
Generating OpenAPI spec...
A `configPath` parameter with a value of /api was provided but no Next REST Framework configs were found.
```

Havent seen any open questions or docs covering this so, **open to any and all ideas!**

_Ps: this is working perfectly fine for the macOS and linux folks_

**Comment by WillGarman** [comment]
I ended up downloading WSL2 (linux distro for windows [guide here](https://www.youtube.com/watch?v=bRW5r7TK6KM)) 

It is working. But still was not working before i started a linux distro

**Comment by blomqma** [comment]
Ah, classic. I did some testing on Windows platform and was able to reproduce this. Basically some dynamic file imports were not working as the file URLs were not resolved correctly on Windows systems causing the CLI commands not to work. This has been addressed in `v5.1.0` so I kindly ask you to upgrade to that version and see if you're problem is gone, thanks for reporting the problem.
